# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/d3d12/appveyor.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/d3d12/appveyor.yml b/third_party/rust/d3d12/appveyor.yml
--- a/third_party/rust/d3d12/appveyor.yml
+++ b/third_party/rust/d3d12/appveyor.yml
@@ -1,29 +1,29 @@
-skip_branch_with_pr: true
-branches:
-  except:
-    - staging.tmp
-environment:
-  global:
-    PATH: '%PATH%;C:\msys64\mingw64\bin;C:\msys64\usr\bin;%USERPROFILE%\.cargo\bin'
-    RUST_BACKTRACE: full
-  matrix:
-  - CHANNEL: stable
-    TARGET: x86_64-pc-windows-msvc
-
-skip_commits:
-  files:
-    - bors.toml
-    - '*.md'
-
-install:
-  - curl -sSf -o rustup-init.exe https://win.rustup.rs
-  - rustup-init -yv --default-toolchain %CHANNEL% --default-host %TARGET%
-  - rustc -vV
-  - cargo -vV
-
-build: false
-test_script:
-  - cargo check
-  - cargo check --features libloading
-  - cargo check --features implicit-link
-  - cargo check --all-features
+skip_branch_with_pr: true
+branches:
+  except:
+    - staging.tmp
+environment:
+  global:
+    PATH: '%PATH%;C:\msys64\mingw64\bin;C:\msys64\usr\bin;%USERPROFILE%\.cargo\bin'
+    RUST_BACKTRACE: full
+  matrix:
+  - CHANNEL: stable
+    TARGET: x86_64-pc-windows-msvc
+
+skip_commits:
+  files:
+    - bors.toml
+    - '*.md'
+
+install:
+  - curl -sSf -o rustup-init.exe https://win.rustup.rs
+  - rustup-init -yv --default-toolchain %CHANNEL% --default-host %TARGET%
+  - rustc -vV
+  - cargo -vV
+
+build: false
+test_script:
+  - cargo check
+  - cargo check --features libloading
+  - cargo check --features implicit-link
+  - cargo check --all-features